.vsts-ci/templates/ci-general.yml

```diff
@@ -62,19 +55,32 @@ steps:
     version: 6.0.x
     performMultiLevelLookup: true
 
+# The build script is always run with PowerShell Core
 - task: PowerShell@2
-  displayName: Build and test
+  displayName: Build and package
   inputs:
     targetType: inline
     script: |
-      Get-ChildItem env:
-      Get-Module -ListAvailable Pester
       Install-Module InvokeBuild -Scope CurrentUser -Force
       Install-Module platyPS -Scope CurrentUser -Force
-      Invoke-Build -Configuration Release
+      Invoke-Build -Configuration Release Package
       $PackageJson = Get-Content -Raw package.json | ConvertFrom-Json
       Write-Host "##vso[task.setvariable variable=vsixPath]$(Resolve-Path powershell-$($PackageJson.version).vsix)"
     workingDirectory: $(Build.SourcesDirectory)/vscode-powershell
+    pwsh: true
```

Comment:
smart lol

Reply:
Turns out that the GitHub PowerShell action defaults to 5.1 on Windows and 7 elsewhere 😳